test/opf/opf.jl

```diff
@@ -69,9 +70,13 @@ include("quad_obj.jl")
 
     @testset _test_socwr_DualFeasibility()
 
-    @testset _test_sdpwrm_DualSolFormat()
+    @testset _test_sdpwrm_DualSolFormat(PGLearn.SDPOPF)
 
-    @testset _test_sdpwrm_DualFeasibility()
+    @testset _test_sdpwrm_DualFeasibility(PGLearn.SDPOPF)
+
+    @testset _test_sdpwrm_DualSolFormat(PGLearn.SparseSDPOPF)
+
+    @testset _test_sdpwrm_DualFeasibility(PGLearn.SparseSDPOPF)
```

Comment:
Can you group those test together so that the visual structure matches which tests are being executed please?
No need to change any internal code, just re-ordering the lines and addling line break to separate groups of tests visually